ansible-chatbot-run.yaml

```diff
@@ -35,8 +35,8 @@ providers:
     config:
       excluded_categories: []
   agents:
-  - provider_id: meta-reference
-    provider_type: inline::meta-reference
+  - provider_id: lightspeed_inline_agent
```

Comment:
You'll probably have to copy the provider into `/.llama/providers.d` too?

https://github.com/ansible/ansible-chatbot-stack/blob/main/Makefile#L60-L61